src/app/components/admin/admin.component.ts

```diff
@@ -13,25 +15,59 @@ import { Router } from '@angular/router';
 
 export class AdminComponent implements OnInit {
   /**
-   * @constructor 
-   * @param router Provides an instance of a router 
+   * @constructor
+   * @param router Provides an instance of a router
    */
-  constructor(private router: Router) { }
+  constructor(private router: Router, private adminservice: AdminService) { }
+
 
-  /**
-   * This is an OnInit function that is evoked once.
-   */
 
+   listofUsers: User[];
+
+   truthy: string = 'btn btn-success';
+   falsy: string = 'btn btn-danger';
+   searchText;
   ngOnInit() {
+
+    this.adminservice.showAllUser()
+    .subscribe(
+      data=> {
+        this.listofUsers = data;
+      }
+    )
+
```

Comment:
@jasonyang1992  this functionality that is already found in the user service; can we consolidate this implementation into one service?